### ISSUE TITLE
Significant change mode

### ISSUE THREAD

**Issue by ackleyimproved** [owntracks/android#1786]
* **App build number**:2.5.0
* **Android version**: 13
* **Device**: Oppo Find X5 Lite
* **Installation source**: Google Play

Ever since the latest version came out significant change mode no longer works. It appears only if the app is active (screen on) it works. Move mode works fine when screen is off. I previously was using certificates however now have reverted to non TLS to get it working again. 

I have minimal location displacement set to 5m, location interval 5s, location interval (move mode) 5s, inaccurate locations set to 30m. I have seen this all working fine in the previous android version.

Allow background activity enabled. Pause app activity if unused to off, No permissions denied. I have tried a few settings changes over the last few weeks but no luck.

It appears that app is not getting GPS updates while phone screen is off.

**Comment by jpmens** [comment]
I think this is a duplicate of https://github.com/owntracks/android/issues/1759

**Comment by ackleyimproved** [comment]
Still getting the issue. I have location settings to allow always as well as above settings.

Also I have uninstalled and re-installed it. 

Anyway I can revert back to the old version for testing?

**Comment by growse** [comment]
APKs are available on the github releases.

**Comment by ackleyimproved** [comment]
Apologies, I re-read the documentation for significant change mode. Appears limitation of android apps operating in background is that only gps updates are every 15 min or rely on a app (such as Google maps) to run in foreground to re-use the gps data from it. It is not possible to have points recorded every 5s unless run in foreground or rely on there app.

What was happening with me (before I thought it broke) is I had android auto running as well, this is how I was able get updates every few metres. I don't have that luxury now since I changed car 

What I wanted was to just take my phone and have it

**Comment by growse** [comment]
Feels like move mode might give you what you need?

**Comment by ackleyimproved** [comment]
> Feels like move mode might give you what you need?

Yes but you end us with thousands of points in same place if you forget to change it back to manual mode.

**Comment by jpmens** [comment]
> What I wanted [..]  have it record every 5 metres or so without any intervention.

Not sure what you want. Significant changes is too little, and when Move mode is suggested it's too much.

You might consider an external app such as Tasker to switch from Significant to Move whenever you want it to.

**Comment by wir3z** [comment]
> It appears that app is not getting GPS updates while phone screen is off.
Are you comparing the same type of travel to previous versions?  My experience all the way back to 2.4.12 was it I was driving, I'd get predicable GPS locations.  If I was walking/biking, then it would ping-pong locations from wifi/cell locations to reduce power.

Cannot comment if having Google Maps open does anything -- haven't tried that.

**Comment by jhuebert** [comment]
I was having problems with this mode too. The solution I found for mine was making sure the location permissions were set properly. When I went into the location permissions for OwnTracks, it was set to `Allow only while using the app` instead of `Allow all the time`. After making that change I haven't had any issues.
